### DISCUSSION TITLE
I get an error Type 'AntMediaClient' does not conform to protocol 'WebSocketDelegate' when implemented the IOS SDK

### DISCUSSION THREAD

**Discussion by sanazaheer1400** [ant-media/Ant-Media-Server#5681] [General]
I tried to implement  [https://antmedia.io/build-and-try-webrtc-ios-app-for-dummies/](url) this tutorial for the IOS app but when I run the app I got 2 errors Type 'AntMediaClient' does not conform to protocol 'WebSocketDelegate' and Type 'ConferenceClient' does not conform to protocol 'WebSocketDelegate'. I use Xcode 15 and IOS 17.0. How can I resolve this issue. Please guide me.

**Comment by yashtandon113** [comment]
Hi @sanazaheer1400 
Thank you for bringing this up.
It is a known issue, and we have already created a GitHub issue for it. You can find more information [here](https://github.com/ant-media/Ant-Media-Server/issues/5565). 
It has already been scheduled and assigned to the developer.
